### DISCUSSION TITLE
Is there a more in depth tutorial on how to run every bare metal?

### DISCUSSION THREAD

**Discussion by NickDatLe** [fastapi/full-stack-fastapi-template#1216] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
n/a
```

### Description

Hi there - is there a more in depth tutorial on how to run everything bare metal? Like run the backend fastapi through VSCode (so I can debug) and run the frontend with react to hit the backend. I've been able to run everything through docker-compose, but I do not believe that is good for development.

I can still use the docker containers for postgresql, but I'm not too sure where Traefik would fit in for a dev environment.

Yes, I tried searching on Google, and I've asked ChatGPT for the answer but I did not find an acceptable answer.

Thank you for your help!

### Operating System

macOS

### Operating System Details

I'm on MacOS m1, running docker desktop.

### Python Version

3.11

**Comment by YuriiMotov** [comment, marked as answer]
I think this part of docs should answer this question: https://github.com/fastapi/full-stack-fastapi-template/blob/master/development.md#local-development

In short:
* run stack with docker compose
* stop the backend and frontend containers
* run backend and frontend servers locally
